NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestQueryStringLayoutRenderer.cs

```diff
@@ -41,12 +40,15 @@ public class AspNetQueryStringLayoutRenderer : AspNetLayoutMultiValueRendererBas
         /// <param name="logEvent"></param>
         protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
         {
-            var httpRequest = HttpContextAccessor?.HttpContext?.TryGetRequest();
+            var httpRequest = HttpContextAccessor.HttpContext.TryGetRequest();
             if (httpRequest == null)
+            {
+                Common.InternalLogger.Debug("aspnet-request-querystring - HttpContext Request is null");
```

Comment:
this is now heavily duplicated. Can't be do this in the base class?

Reply:
I guess it should be possible. Actually it could be nice if there was only one message for a single logevent. Right now one will get a debug-message for every triggered layout-renderer.

Reply:
I can live with the duplicate message, but not really with the duplicate code ;)

Reply:
Would it be okay if I just remove the debug-messages then. Because I find it difficult to modify these layoutrenderers without it being breaking interface changes.

Reply:
And they are gone :)